src/ui/main_window.py

```
@@ -808,71 +1005,149 @@ def progress_callback(completed: int, total: int, message: str) -> bool:
+        new_arr = ((new_arr - 0.5) * contrast_factor + 0.5) * exposure_factor
+        new_arr = np.clip(new_arr, 0, 1)
+        
+        return new_arr
+
+    def _apply_exposure_contrast_global(
+        self, 
+        arr: np.ndarray, 
+        exposure: int, 
+        contrast: int,
+        global_min: float,
+        global_max: float,
```

Ruff (0.14.3)

Unused method argument: `global_max`

(ARG002)